lib/ignored-paths.js

```diff
@@ -21,42 +22,18 @@ debug = debug("eslint:ignored-paths");
 //------------------------------------------------------------------------------
 
 var ESLINT_IGNORE_FILENAME = ".eslintignore";
+var DEFAULT_IGNORE_PATTERNS = [
+    "/node_modules/",
+    ".git/",
```

Comment:
Can't we just ignore all dot-directories and files by default? Otherwise, this becomes a maintenance issue as we need to keep updating.